docs/staging-use-cases.md

```diff
@@ -131,6 +131,23 @@ Multitenancy Management
 ### Adoption
 - 🔭 As an operator, given I have created a Kubernetes-conformant cluster without ClusterAPI, I want to use ClusterAPI to manage it. In order to do so, I need to know the requirements for adopting/importing this cluster in terms of required CRD’s and operators (e.g Machine and Cluster objects).
 
+### Cluster Health Checking
+
+Cluster Health Checking is a service to provide the health status of Kubernetes cluster and its components.
+
+- As an operator, given I have created a Kubernetes-conformant cluster with ClusterAPI, I want to check the Kubernetes cluster node status.
+  - Describe nodes and provide details if they are ready/healthy or not ready/healthy.
+  - List conditions for any nodes which are `NotReady`, list information about allocated resources.
+
+- As an operator, given I have created a Kubernetes-conformant cluster with ClusterAPI, I want to check the kube-apiserver status.
+
+- As an operator, given I have created a Kubernetes-conformant cluster with ClusterAPI, I want to check the Kubernetes core components status, like etcd, ingress controller, other add-on components etc.
+
+- As an operator, given I have created a Kubernetes-conformant cluster with ClusterAPI, I want to check unhealthy Pods statuses in configured namespace.
+  - Provide the details on any pods which are unhealthy in `kube-system` namespace. Filter the unhealthy pods for their status(`kubectl get pods --show-labels -n kube-system | grep -vE "Running|Completed"`)
+  - Describe any pods which are not `Completed|Running`, list the Events to provide hints on the failure.
```

Comment:
please uppercase `pods` -> `Pods` in the new text. multiple instances above.

Reply:
Done